### PR TITLE
Improve quantileSeq typings

### PR DESCRIPTION
**Pull request by domdomegg**
Fixes https://github.com/josdejong/mathjs/issues/3197

### CODE REVIEW

File: types/index.d.ts [hunk 1]
```diff
@@ -2655,7 +2655,7 @@ export interface MathJsInstance extends MathJsFactory {
    */
   max(...args: MathScalarType[]): MathScalarType
   /**
-   * @param A A single matrix
+   * @param A An array of scalar values
```
Comment (josdejong) [inline types/index.d.ts:2658]: The argument can be either an array with scalar values (a 1-D matrix) or an array with arrays with scalar values (a 2-D matrix). I prefer to either keep these descriptions as they where, "A single matrix", or turn it into something like "A matrix" or "A 1-D or 2-D array with scalar values"

File: types/index.d.ts [hunk 2]
```diff
@@ -2805,6 +2805,19 @@ export interface MathJsInstance extends MathJsFactory {
...
 
+  /**
+   * @param A An array of scalar values
+   * @param probOrN prob is the order of the quantile, while N is the
+   * amount of evenly distributed steps of probabilities; only one of
+   * these options can be provided
+   * @param sorted =false is data sorted in ascending order
+   * @returns Quantile(s)
+   */
+  quantileSeq<T extends MathScalarType>(
```
Comment (josdejong) [inline types/index.d.ts:2816]: Thanks, this looks good! Can you add a few cases to [/test/typescript-tests/testTypes.ts](https://github.com/josdejong/mathjs/blob/develop/test/typescript-tests/testTypes.ts) to ensure the new typings actually work?

**Review by josdejong — COMMENTED** [review]
Thanks Adam! I made two inline comments, can you have a look at those?

File: types/index.d.ts (hunk 2 above)
Comment (domdomegg) [reply types/index.d.ts:2816]: Sure, have done ✅

Ready for re-review

Comment (josdejong) [reply types/index.d.ts:2816]: Thanks 👌

**Comment by josdejong** [comment]
Thanks for the updates, this looks good. 

I see the build is still throwing 1 TypeScript error: 

```
Error: test/typescript-tests/testTypes.ts(2539,10): error TS2554: Expected 2-3 arguments, but got 1.
```

Can you resolve that? I think you have to specify a probability in the test on that line. Besides that all is ready to merge.

**Comment by domdomegg** [comment]
Whoops, missed that! Thanks for flagging, `npm run test:types` should now work (didn't realise this wasn't run by `npm test`).

**Comment by josdejong** [comment]
Thanks. Good point about documentation missing on how to run these tests. I've added that via 49df889.

**Comment by josdejong** [comment]
The fix is published now in `v12.4.3`

**Pull request merged**